aws-athena/views/model-vw_card_res_input.sql

```diff
@@ -415,6 +415,17 @@ SELECT
             nn1.prox_avg_school_rating_in_half_mile IS NULL
             THEN nn2.prox_avg_school_rating_in_half_mile
     END AS prox_avg_school_rating_in_half_mile,
+    CASE --new, not sure case statement is necessary
```

Comment:
```suggestion
    CASE
```

The case statement here is necessary to fill in any missing values from the nearest neighbor. It's unlikely to be used in this case but doesn't hurt.